client/starwhale/api/_impl/data_store.py

```diff
@@ -48,10 +51,12 @@
 except ImportError:
     has_fcntl = False
 
+datastore_table_file_ext = ".swds.json"
```

Comment:
`swds` = `starwhale dataset` or `starwhale datastore` ? 

`swds` has already been used for dataset.

Reply:
changed to `.sw-datastore.json`